hw/ip/otbn/dv/otbnsim/BUILD

```diff
@@ -0,0 +1,16 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@rules_python//python:defs.bzl", "py_binary")
+
+package(default_visibility = ["//visibility:public"])
+
+# Make standalone simulator visible to OTBN tests within Bazel (see
+# rules/otbn.bzl).
+exports_files(["standalone.py"])
+
+# py_binary(
```

Comment:
Is there a reason this is commented out?

Reply:
Forgot to remove it, thanks! Fixed.